source/materials/MaterialsList.cc

```diff
@@ -121,21 +150,29 @@ namespace materials {
         kStateGas, temperature, pressure);
 
 
-      G4Element* Xe = new G4Element("GXeDepleted", "Xe", 5);
-
+      G4Element* Xe = new G4Element("GXeDepleted", "Xe", 9);
+      
+      G4Isotope* Xe124 = new G4Isotope("Xe129", 54, 124, XenonMassPerMole(124));
```

Comment:
The isotopic number is not the same in the string and in the other arguments. Also, you did change the previous values. Can you point to a reference for that?